join/v0.2/join-v0.2.graphql

```diff
@@ -0,0 +1,25 @@
+enum Graph
+
+directive @type(
+  graph: Graph!,
+  key: String!,
+  extension: Boolean!
```

Comment:
Fwiw, `extension` has a default value of `false` and `resolvable` a default value of `true`.

Reply:
done